Content.Server/_Shitmed/DelayedDeath/DelayedDeathSystem.cs

```diff
@@ -40,6 +43,19 @@ public override void Update(float frameTime)
                 // go crit then dead so deathgasp can happen
                 _mobState.ChangeMobState(ent, MobState.Critical, mob);
                 _mobState.ChangeMobState(ent, MobState.Dead, mob);
+
+                // goob code
+                var ev = new DelayedDeathEvent(ent);
+                RaiseLocalEvent(ent, ref ev);
+
+                if (ev.Cancelled)
+                {
+                    RemComp<DelayedDeathComponent>(ent);
```

Comment:
```suggestion
                    RemCompDeferred(ent, comp);
```
RemComp spotted in while loop.